main/client/header/navbar.js

```diff
@@ -0,0 +1,11 @@
+Meteor.startup(function() {
+
+  // Adds class .shrinked to #top-nav
+  $(window).scroll(function() {
+    let topNav = $('nav#top-nav');
+    if ($(document).scrollTop() > 75) topNav.addClass('shrinked');
```

Comment:
Sure, I'll try to make better comments. What's the procedure to update a pull request after code review? Should I just keep commiting to the same branch?